mkt/constants/regions.py

```diff
@@ -108,7 +125,7 @@ class RESTOFWORLD(REGION):
     if country.get('ratingsbody'):
         country['ratingsbody'] = getattr(ratingsbodies, country['ratingsbody'])
 
-    globals()[k] = type(k, (REGION,), country)
+    locals()[k] = type(k, (REGION,), country)
```

Comment:
AFAIK the `globals` call was added when marketplace-constants was first used to pull the countries, in c8de04187e84c27ae0eb66fbe945a01d720cb3f2 ; I didn't see any changes to `locals` and back in the log.

I don't really see a good reason to use `globals` here, that's why I changed it. @andymckay maybe you know ?